client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -212,8 +227,25 @@ export default React.createClass( {
 				onDialogClose={ this.closeDialog }
 				onDialogOpen={ this.openDialog }
 				onDialogSelect={ this.handlePrivacyDialogSelect }
-				isDialogVisible={ this.state.isDialogVisible }
-				productsList={ this.props.productsList }/>
+				isDialogVisible={ this.state.isPrivacyDialogVisible }
+				productsList={ this.props.productsList } />
+		);
+	},
+
+	handleFrSubmit( registrantExtraInfo ) {
+		this.setState( { registrantExtraInfo } );
+		this.closeDialog( 'fr' );
```

Comment:
I think we never open a dialog named `fr`. Did you mean `isFrDialogVisible`?
I think it'd would make it more resilient to just create functions for every dialog (or use const) instead of relying on literal strings sprinkled everywhere.